src/Layouts/Partials/BorderItem.vala

```diff
@@ -191,20 +203,38 @@ public class Akira.Layouts.Partials.BorderItem : Gtk.Grid {
         color_popover.add (color_picker);
 
         attach (color_chooser, 0, 0, 1, 1);
-        attach (hidden_button, 1, 0, 1, 1);
-        attach (delete_button, 2, 0, 1, 1);
+        attach (eyedropper_button, 1, 0, 1, 1);
+        attach (hidden_button, 2, 0, 1, 1);
+        attach (delete_button, 3, 0, 1, 1);
 
         set_color_chooser_color ();
         set_button_color ();
     }
 
     private void create_event_bindings () {
+        eyedropper_button.clicked.connect (on_eyedropper_click);
         delete_button.clicked.connect (on_delete_item);
         hidden_button.clicked.connect (toggle_visibility);
         model.notify.connect (on_model_changed);
         color_chooser_widget.notify["rgba"].connect (on_color_changed);
     }
 
+    private void on_eyedropper_click () {
+        eyedropper = new Akira.Utils.ColorPicker ();
+        eyedropper.show_all ();
+
+        eyedropper.picked.connect ((picked_color) => {
+            alpha = ((int)(picked_color.alpha * 255));
+            color_chooser_widget.set_rgba (picked_color);
+            on_color_changed ();
```

Comment:
This is not necessary since the changes of the RGBA of the color chooser is binded to that method.